scripts/st2bootstrap-el7.template.sh

```diff
@@ -158,6 +151,21 @@ install_st2_dependencies() {
   fi
 }
 
+install_rabbitmq() {
+  sudo yum -y install curl rabbitmq-server
+
+  # Configure RabbitMQ to listen on localhost only
+  sudo sh -c 'echo "RABBITMQ_NODE_IP_ADDRESS=127.0.0.1" >> /etc/rabbitmq/rabbitmq-env.conf'
+
+  sudo systemctl start rabbitmq-server
+  sudo systemctl enable rabbitmq-server
+
+  sudo rabbitmqctl add_user stanley "${ST2_RABBITMQ_PASSWORD}"
```

Comment:
Can you change the username from `stanley` to `stackstorm`?  This is to be consistent with the username that we used for MongoDB.